docs/docs/migrations.md

```diff
@@ -306,6 +306,74 @@ queryInterface.removeIndex('Person', 'SuperDuperIndex')
 queryInterface.removeIndex('Person', ['firstname', 'lastname'])
 ```
 
+### addConstraint(tableName, indexNameOrAttributes, options)
+This method adds a new constraint of the specified type.
```

Comment:
In the description explain meaning of arguments, and what common options we have. Here you can explain things like `options.name` and what happens if you dont pass them